site.yml

```diff
@@ -292,6 +292,22 @@
       tags: ['openstack', 'horizon', 'control']
   environment: "{{ env_vars|default({}) }}"
 
+- name: install swift-common
+  hosts: swiftnode
+  any_errors_fatal: true
+  roles:
+    - role: swift-common
+      tags: ['openstack', 'swift', 'swift-common', 'data']
+  environment: "{{ env_vars|default({}) }}"
+
+- name: swift bootstrap rings
```

Comment:
Although, swift-ring appears to only run on swiftnode-primary, whereas the play above runs swift-common across all the swift nodes.

Without the above play, swifnode_primary would get swift-common, then swift-ring. Then all the swift nodes would get swift-common via the swift-object role in the `swift code and config` role.

We're bootstrapping the rings before doing the code and config play, and the bootstrap uses swiftool, which I believe will contact the other swift nodes and expect some scaffolding to be in place. Perhaps that's why Craig moved the common role to be applied before the ring bootstrap.